superset/config.py

```diff
@@ -263,7 +263,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 # Grant public role the same set of permissions as for a selected builtin role.
 # This is useful if one wants to enable anonymous users to view
 # dashboards. Explicit grant on specific datasets is still required.
-PUBLIC_ROLE_LIKE = "Gamma"
+PUBLIC_ROLE_LIKE = None
```

Comment:
I assume we should add the optional type, too:
```suggestion
PUBLIC_ROLE_LIKE: Optional[str] = None
```